src/bundle/Resources/views/content/content_preview.html.twig

```diff
@@ -42,7 +42,7 @@
                 <div class="ez-preview__item ez-preview__item--siteaccess">
                     <select class="form-control">
                         {% for previewSiteaccess in siteaccesses %}
-                            <option value="{{ path('_ezpublishPreviewContent', {'contentId': content.id, 'versionNo': versionNo, 'language': language_code, 'siteAccessName': previewSiteaccess}) }}">{{ previewSiteaccess }}</option>
+                            <option value="{{ path('_ezpublishPreviewContent', {'contentId': content.id, 'versionNo': versionNo, 'language': language_code, 'siteAccessName': previewSiteaccess}) }}">{{ ezplatform_admin_ui_site_access_name(previewSiteaccess) ~ ' (' ~ previewSiteaccess ~ ')' }}</option>
```

Comment:
I don't think the label should have format: `Readable name (identifier)` but simply `Readable name`. Can you double check that with wireframes?

Reply:
Based on:
https://jira.ez.no/browse/EZP-28806?focusedCommentId=222740&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-222740